Resources/public/js/views/services/ez-locationviewviewservice.js

```diff
@@ -43,6 +44,110 @@ YUI.add('ez-locationviewviewservice', function (Y) {
         },
 
         /**
+         * _sendContentToTrashConfirmBox event handler on send to trash button
```

Comment:
Should be something like: "`sendToTrashAction` event handler, it asks confirmation to the user before sending the location to the trash."